config/locales/en/pvb2-mailers.yml

```diff
@@ -158,36 +158,45 @@ en:
       subject: "CANCELLED: Your %{prison_name} prison visit for %{date}"
       salutation: Dear %{name},
       intro:
-        We're sorry but your visit to to %{prisoner} at %{prison} on %{date}
-        has been cancelled
-      slot_unavailable: >-
-        because the visit time is no longer available.
-      visitor_banned_html: |
-        <p>We're sorry but your visit to to %{prisoner} at %{prison} on %{date}
-        has been cancelled.</p>
+        We're sorry to let you know we have cancelled your visit to %{prisoner} at %{prison} on %{date}.
+      slot_unavailable_html: >-
+        <p>We’re sorry but the slot you chose is no longer available.</p>
```

Comment:
Suggest we amend the copy from 'slot' to 'date and time' as this will make more sense to the visitor.